xds/src/main/java/io/grpc/xds/ClientXdsClient.java

```diff
@@ -473,8 +469,8 @@ static void validateCommonTlsContext(
           "common-tls-context with validation_context_certificate_provider_instance is not"
               + " supported");
     }
-    String certInstanceName = null;
-    if (!commonTlsContext.hasTlsCertificateCertificateProviderInstance()) {
+    String certInstanceName = getIdentityCertInstanceName(commonTlsContext);
+    if (certInstanceName == null) {
```

Comment:
The error messages `"common-tls-context with tls_certificates is not supported"`,  and `"common-tls-context with tls_certificate_sds_secret_configs is not supported"` seems misleading. Should mention `TlsCertificateProviderInstance` is unset.

Reply:
fixed.